force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +304,14 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
+        Book b = new Book('A Brief History of Time', 'Stephen Hawking');
+
 
         // Call the getBookDetails method on the book object to get the details of the book
+        String bDetails = b.getBookDetails();
 
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return bDetails; // Replace null with the variable you used to store the result
```

Comment:
Can you solve this with one line of code? I think you can and not lose readability.